PCbuild/_freeze_module.vcxproj

```diff
@@ -374,6 +374,33 @@
       <OutFile>$(PySourcePath)Python\frozen_modules\getpath.h</OutFile>
     </GetPath>
   </ItemGroup>
+        <ItemGroup>
+          <!-- BEGIN freeze mappings -->
```

Comment:
It would be nice to fix the indentation on these lines. I don't see where it is though.

Otherwise anyone who edits this file manually is likely to fix it and trigger regeneration.